CHANGES.rst

```diff
@@ -1,7 +1,10 @@
 0.3.2 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Change the name of the package to `sqlalchemy_redshift` to match the naming
+  convention for other dialects; the `redshift_sqlalchemy` now references
```

Comment:
missed a word? Should this be "the deprecated `redshift_sqlalchemy` now references..."